gatsby/content/docs/online-events.mdx

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: Running online events with Open Source communication tools
+section: Introduction
+slug: running-online-events
+---
+
+Starting in 2020, team Matrix planned to start running regular meetup events, 
+with speakers, talks, Q+A, beer, pizza etc.
+
+We hosted one, in London, in February. Around that time and immediately 
+afterward, much of the world shut down due to a global pandemic, and hosting 
+in-person meetups became infeasible.
+
+Rather than drop the idea, we decided to make use of the technologies we 
+regularly promote to host a *virtual* meetup event. The idea being to try to 
+recreate the fun and atmosphere of a real meetup in a decentralised way.
+
+## Use Jitsi for Video
+
+It's important to us, wherever possible, to use Open tools. Jitsi Meet is Open 
+Source, self-hostable video conferencing platform.
+[Installing Jitsi is a much-documented experience](https://github.com/jitsi/jitsi-meet/blob/master/doc/
+quick-install.md) that I won't delve into here, but for our purpose we used our 
+installation at <https://jitsi.modular.im>.
+
+First, we want to get our speakers all connected. To do this, we created a new 
+room on Jitsi (the name doesn't matter), and made sure to add a password, since 
+this is not the default. All the speakers for the session will connect to this 
+room.
+
+![add a password to a jitsi room](/docs/img/jitsi-add-password.png)
+
+Next, we configured the Jitsi Meet server to stream to YouTube, and also made 
+the stream available directly via a video control on https://stream.matrix.org.
```

Comment:
I feel like this is going to make a lot of people ask about how we did that, so probably worth elaborating on that point?

Reply:
I agree. For now I'm waiting on time being available from the person or people who set it up.